aim/utils/tracking.py

```diff
@@ -68,7 +64,13 @@ def initialize(self) -> None:
         if self.initialized:
             return
 
+        segment_logger = logging.getLogger('segment')
+        segment_logger.disabled = True
```

Comment:
why do we disable the segment logger?

Reply:
Even with the `logLevel == ERROR` segment might output errors. For example in the mentioned case `flush()` will output the timeout errors.
With the logLevel `INFO` the logs get flooded.